source/sdk/lang/equalities.ooc

```diff
@@ -0,0 +1,47 @@
+// The following functions were moved from HashMap.ooc
```

Comment:
I think the filename should have a capital `E`, but even then, wouldn't `Equals` make more sense?